### PR TITLE
fix: Vector drawables for icons

### PR DESCRIPTION
**Pull request by immadisairaj**
First steps for issues #1381 

Changes: 
Replaced pngs with vector drawables for letter_s and baseline_star_rate_black
changed android:src to app:srcCompat in xml files

When converting following files to vector the file will be having more paths due to colors and the application crashes if added these files as vector.
intro1.png, intro2.png, intro3.png, intro4.png

### CODE REVIEW

**Comment by immadisairaj** [comment]
@iamareebjamal 
Changed as requested

File: app/src/main/res/drawable/letter_s.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="57dp" android:viewportHeight="56.748768"
```
Comment (iamareebjamal) [inline app/src/main/res/drawable/letter_s.xml]: What about this?

**Comment by immadisairaj** [comment]
@iamareebjamal 
Resolved changes. Can you please look into it..

**Comment by immadisairaj** [comment]
Can I know why this is not merging?

**Comment by batbrain7** [comment]
@immadisairaj the mentors are busy most of the times with their daily lives, once a PR is approved it will get merged sooner or later.

**Pull request merged**